libfwupdplugin/fu-uefi-device.c

```diff
@@ -0,0 +1,422 @@
+/*
+ * Copyright 2025 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#define G_LOG_DOMAIN "FuUefiDevice"
+
+#include "config.h"
+
+#include "fu-bytes.h"
+#include "fu-device-event-private.h"
+#include "fu-device-private.h"
+#include "fu-uefi-device-private.h"
+
+/**
+ * FuUefiDevice:
+ *
+ * A UEFI key, typically only available on Linux and FreeBSD.
```

Comment:
/cough
https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-setfirmwareenvironmentvariablea
https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getfirmwareenvironmentvariablea

It would be REALLY cool if dbx got updatable from Windows using fwupd....

Reply:
Ya, that would really open the floodgates for QA tho.

Reply:
I fixed the header comment -- we have a fu-windows-efivars.c so in theory it might work.